### ISSUE TITLE
Examples not working properly with Oculus

### ISSUE THREAD

**Issue by RickBlacker** [thestonefox/VRTK#1844]
### Environment

 * Source of VRTK: I think Github, been a while since I downloaded it.
 * Version of VRTK: 3.3
 * Version of Unity3D: 2018.1.0f2
 * Hardware used (e.g. Vive/Oculus): Oculus
 * SDK Oculus Utilities 1.27

### Steps to reproduce
I loaded up the constructor scene. Then I opened up the straight pointer scene.  

### Expected behavior

Thought pointer would follow the hand position.

### Current behavior

The pointers appear when I touch the top of the Oculus joystick. They do not follow the controllers, rather appear to be coming out from behind me, straight at the robot.   The same happens when I try the BezierPointer example.  The pointers seem to be coming out from behind my head.

Are there specific versions that we need to be using? 

Vive seems to be working Ok.

**Comment by thestonefox** [comment]
Have a look at this issue

https://github.com/thestonefox/VRTK/issues/1819

**Comment by bddckr** [comment]
Constructor scenes is examples only and has been removed in the latest master AFAIK.

Warnings about missing scripts in the example scenes come from you not importing all supported SDKs (the example scenes are set up to work with _everything_). Ignore them in there but set up your SDK Manager properly in your own project.

For additional help join [![Slack](http://sysdia2.co.uk/badge.svg)](http://invite.vrtk.io) because you can instantly get answers instead of waiting on here. It's also way easier for troubleshooting in general. **GitHub issues are only useful for proper bug reports with steps to reproduce in an example scene like the Issue Template requires.**

**Comment by RickBlacker** [comment]
Hi bddckr 
I've tried to get help on Slack, not getting a lot of attention on my post. I'm really sorry for continuing to post here.   

I propably should have have said that I have been working on an app using VRTK.  I have my app working with Vive.  But, when I plugged in my Oculus, that's when I started to see odd behavior.  Which is why I started trying to look at the sample scenes.  In both my app and the example scenes, the Oculus controllers are not tracking properly. I have ensured that my Oculus is setup properly and ran a different Oculus app. Seemed to work fine.  

Is there possibly a setting, or a method to configure Oculus that I may have missed?  Or am I wrong in my assumption that I can unplug my Vive, plug in my Oculus and the example scene ( straight pointer ) should just work without any modifications?

From my understanding I have everything I need. 
- Version of VRTK: 3.3
- Version of Unity3D: 2018.1.0f2
- Hardware used (e.g. Vive/Oculus): Oculus
- SDK Oculus Utilities 1.27

I greatly appreciate your help!!!
